index.js

```diff
@@ -7,7 +7,9 @@ module.exports = {
             save: 'Lưu',
             create: 'Tạo mới',
             edit: 'Sửa',
+            export: 'Xuất khẩu',
```

Comment:
I think just 'Xuất' more make seen in this case. Because usually 'Xuất khẩu' for product.